### ISSUE TITLE
Bank Storage / Tank Storage

### ISSUE THREAD

**Issue by purrloftruth** [EnigmaticaModpacks/Enigmatica10#149]
### URL

https://www.curseforge.com/minecraft/mc-mods/bank-storage

### Why would you like the mod added?

(and https://www.curseforge.com/minecraft/mc-mods/tank-storage)

Enigmatica 6 had a mod called Dank Storage which added an item called a "Dank" that stored large quantities of a few types of items (and increasingly higher quantities for higher-tier versions). Bank Storage is a modern mod that behaves very similarly. Tank Storage is another mod by the same developer as Bank Storage which, instead of items, stores fluids.

This kind of item is very helpful when mining, as it allows you to collect large numbers of specific blocks in one place and not fill up your inventory.

**Comment by MuteTiefling** [comment]
Sophisticated backpacks pretty handily covers this niche and adds a lot of other qol on top of it.

**Comment by MuteTiefling** [comment]
AE2 and Just Dire Things both also have features to replicate that remote pull (among several other mods).

The building stuff sounds interesting, though we did also just add Utilitarian that has the quark trowel. Ars Nouveau also has a nice randomizer for placing blocks. 

Out of curiosity, I loaded it up in my instance. It's shockingly cheap for what it does, so it would likely become the defacto storage solution over basically anything else. 

I'm not really sold. But I'll let others weigh in.
